config/services.yml

```diff
@@ -8,14 +8,10 @@ services:
 
     PhpTuf\ComposerStager\:
         resource: '../src/*'
-        exclude:
-            - '../src/Exception'
-
-    PhpTuf\ComposerStager\Console\:
-        resource: '../src/Console/*'
         public: true
```

Comment:
Alas, I had to make all classes public in order to access a specific file copier in functional tests. 😕